internal/live/douyin.go

```
@@ -0,0 +1,491 @@
+			}
+		case num == userSecUID && wire == wireLen:
+			if s, ok := r.LenBytes(); ok {
+				u.SecUID = string(s)
+			}
+		case num == userIPLabel && wire == wireLen:
+			if s, ok := r.LenBytes(); ok {
+				u.IPLabel = string(s)
+			}
+		default:
+			if !r.Skip(wire) {
+				break
```

golangci-lint (2.12.2)

[error] SA4011: ineffective break statement. Did you mean to break out of the outer loop?

(staticcheck)